src/python/example/diagram_vectorizations_distances_kernels.py

```diff
@@ -49,50 +54,61 @@ def pow(n):
 
 CP = ComplexPolynomial(threshold=-1, polynomial_type="T")
 cp = CP.fit_transform(diags)
-print("Complex polynomial is " + str(cp[0,:]))
+print("Complex polynomial is " + str(cp[0, :]))
 
 TV = TopologicalVector(threshold=-1)
 tv = TV.fit_transform(diags)
-print("Topological vector is " + str(tv[0,:]))
+print("Topological vector is " + str(tv[0, :]))
 
-PI = PersistenceImage(bandwidth=.1, weight=lambda x: x[1], im_range=[0,1,0,1], resolution=[100,100])
+PI = PersistenceImage(bandwidth=.1,
+                      weight=lambda x: x[1],
+                      im_range=[0, 1, 0, 1],
+                      resolution=[100, 100])
 pi = PI.fit_transform(diags)
-plt.imshow(np.flip(np.reshape(pi[0], [100,100]), 0))
+plt.imshow(np.flip(np.reshape(pi[0], [100, 100]), 0))
 plt.title("Persistence Image")
 plt.show()
 
 ET = Entropy(mode="scalar")
 et = ET.fit_transform(diags)
-print("Entropy statistic is " + str(et[0,:]))
+print("Entropy statistic is " + str(et[0, :]))
 
 ET = Entropy(mode="vector", normalized=False)
 et = ET.fit_transform(diags)
 plt.plot(et[0])
 plt.title("Entropy function")
 plt.show()
 
-D = np.array([[1.,5.],[3.,6.],[2.,7.]])
+D = np.array([[1., 5.], [3., 6.], [2., 7.]])
 diags2 = [D]
 
-diags2 = DiagramScaler(use=True, scalers=[([0,1], MinMaxScaler())]).fit_transform(diags2)
+diags2 = DiagramScaler(use=True,
+                       scalers=[([0,
+                                  1], MinMaxScaler())]).fit_transform(diags2)
```

Comment:
`yapf3 --style google` avoids the ugly break in the middle of `[0, 1]`, but that appears to be random, making  `scalers` one letter longer makes it break there again.